### ISSUE TITLE
pyright incorrectly flags complex numbers (1j) as errors

### ISSUE THREAD

**Issue by colinRawlings** [microsoft/pyright#477]
Pyright is a wonderful tool but in version 1.1.18 it incorrectly flags statements like:

```python
a = 1 + 1j
```

as syntax errors.  The exact error seems to vary but a common example is: 

"Statements must be separated by newlines or semicolons"

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version.

**Comment by erictraut** [comment]
This support is now included in 1.1.19, which I just published.
